electron/protocol.ts

```diff
@@ -21,53 +19,11 @@ export async function handleProtocol(window : BrowserWindow, url : string) {
   const appName = args.get('appName')
   const game = LegendaryGame.get(appName)
   if (command === 'launch') {
-    const { title, is_installed } = await game.getGameInfo()
+    const { is_installed } = await game.getGameInfo()
     if (!is_installed) {
       console.log(`ProtocolHandler: "${appName}" not installed, ignoring launch request.`)
       return
     }
-    const { response } = await dialog.showMessageBox(window, {
-      'buttons': [
-        'No', 'Yes'
-      ],
-      'cancelId': 0,
-      'message': `Launch ${title}?`,
-      'type': 'question'
-    })
-    if (response === 1) {
-      return await game.launch()
```

Comment:
Hold up, why're we not launching the game from the backend? This introduces an unnecessary dependency on the frontend, which might hurt later since I plan to add options to not lauch the frontend, but instead only the backend. I'll advise leaving a `//TODO` here for future refactors.

Reply:
That's doesn't make sense. There is not launcher that does that.
Both steam and lutris launch themselves. What can we do is to send heroic to the tray.
How can we launch something without user feedback? How can the user know if the game was launched or not?
And if we have an error? Example: authentication expired. The user will be waiting for something that will never happen.

Reply:
I have to agree on that 😛

Reply:
I think we can discuss later on this.
For now we need to merge this to @Nocccer to update the tests for the install functions, etc...